pkg/uefi/flash.go

```diff
@@ -242,8 +242,8 @@ func NewFlashImage(buf []byte) (*FlashImage, error) {
 		return nil, err
 	}
 
-	// FlashRegions
-	frs := f.IFD.Region.FlashRegions
+	// FlashRegions is an array, make a slice to manipulate it
+	frs := f.IFD.Region.FlashRegions[:]
```

Comment:
Do you still need to manipulate it? I don't see anything that changes the size in the following code

Reply:
So maybe the comment is more "make it a lice to avoid duplicating it" ? what do you think ?

Reply:
or better: // FlashRegions is an array, make a slice to keep reference to it's content
which is exactly what we want

Reply:
Either is fine. I missed the original intent of this change. Thanks!